yacht/lib/curatoresclientfactory.py

```diff
@@ -0,0 +1,15 @@
+import os
+from esclientfactory import EsClientFactory
+
+class CuratorEsClientFactory():
```

Comment:
Consider moving this to the `Yacht()` class constructor and passing the ES object to source and create objects.